### ISSUE TITLE
I have a Solution to your email verification not working.

### ISSUE THREAD

**Issue by aadit2004123** [Alan4747/instagram-auto-create-account#58]
Hi!  I love your program and it works really smoothly so hats off to you. I noticed that there was an issue everyone was facing as the email verification was not working. I wanted to suggest if you could try to change the code to work for a website like https://temp-mail.org/en/
Doing this would make your program useable. I tried doing it on my own but you clearly know ur way around this so I was wondering if you could do that?

Thankyouuuu.

**Comment by edisax** [comment]
Hi!
I updated the script to get emails from https://temp-mail.org/en/ but they do not have that many domains so IG have probably flagged those domains cause I'm getting
"The IP address you are using has been flagged as an open proxy. If you believe this to be incorrect...."

Does it work for you if you create one acc manually with it?

**Comment by aadit2004123** [comment]
yeah it works for me lmao... i even coded my own bot to work with it and it worked great...... apart from the whole mobile verification thing.. i cant figure that out lol

**Comment by aadit2004123** [comment]
if you have suggestions for the mobile verification thing, maybe we could talk more, and I could help out with the tempmail thing. I'm not that great at this but I could try and help

**Comment by edisax** [comment]
Yeah got it, well that's the issue with the low amount of domains they have.. Since they can easily flag those domains and then the mobile verification and "IP flagged as proxy" will pop up. If you use a google email you wont get that issues :) 
There is no other way to bypass the mobile verification. You can however buy sms verification services think it costs about 0.2$ per sms. But if can find another site like email-fake.com that has plenty of domains we might solve it that way :)

**Comment by priyaank2004** [comment]
> yeah it works for me lmao... i even coded my own bot to work with it and it worked great...... apart from the whole mobile verification thing.. i cant figure that out lol

bro can me tell me what changes to make in program. plz help  me out. can you share me your code
..i also updated website for email but it shows me error.

I updated that fakemail.py file

import requests
from bs4 import BeautifulSoup

def getFakeMail():
    url = 'https://10minutemail.net/'
    req = requests.get(url)
    soup = BeautifulSoup(req.content, "html.parser")
    mail = soup.find("button", {"id": "copy-button"})
    return mail['data-clipboard-text']

but it shows that below error
 
File "C:\Users\Apple 2.0\Desktop\qqq\botAccountCreate.py", line 72, in
domain = fMail[1]
IndexError: list index out of range

**Comment by aadit2004123** [comment]
i made an entirely new code

**Comment by aadit2004123** [comment]
> Yeah got it, well that's the issue with the low amount of domains they have.. Since they can easily flag those domains and then the mobile verification and "IP flagged as proxy" will pop up. If you use a google email you wont get that issues :) There is no other way to bypass the mobile verification. You can however buy sms verification services think it costs about 0.2$ per sms. But if can find another site like email-fake.com that has plenty of domains we might solve it that way :)

i seeee

**Comment by varocarras** [comment]
@aadit2004123 I would love to help you set this up and work with you. I can have residential proxy service that will work and I can pay for the sms service if we get it to work. Email me at varocarras@gmail.com if you are interested in working together! I know selenium well too
